code/game/turfs/hybrisa.dm

```diff
@@ -0,0 +1,717 @@
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony
+	name = "bare metal walls"
+	icon = 'icons/turf/walls/hybrisa_colonywall.dmi'
+	icon_state = "strata_bare_outpost_"
+	desc = "A thick and chunky metal wall. The surface is barren and imposing."
+	walltype = WALL_STRATA_OUTPOST_BARE
+/turf/closed/wall/hybrisa/colony/reinforced
+	name = "ribbed metal walls"
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/reinforced/hull
+	hull = TRUE
+	icon_state = "strata_hull"
+	desc = "A thick and chunky metal wall that is, just by virtue of its placement and imposing presence, entirely indestructible."
+
+// Hospital
+
+/turf/closed/wall/hybrisa/colony/hospital/ribbed //this guy is our reinforced replacement
+	name = "ribbed metal walls"
+	icon = 'icons/turf/walls/hybrisa_colonywall_hospital.dmi'
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/hospital
+	name = "bare metal walls"
+	icon = 'icons/turf/walls/hybrisa_colonywall_hospital.dmi'
+	icon_state = "strata_bare_outpost_"
+	desc = "A thick and chunky metal wall. The surface is barren and imposing."
+	walltype = WALL_STRATA_OUTPOST_BARE
+/turf/closed/wall/hybrisa/colony/hospital/reinforced
+	name = "ribbed metal walls"
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/hospital/reinforced/hull
+	hull = TRUE
+	icon_state = "strata_hull"
+	desc = "A thick and chunky metal wall that is, just by virtue of its placement and imposing presence, entirely indestructible."
+
+// Offices
+
+/turf/closed/wall/hybrisa/colony/office/ribbed //this guy is our reinforced replacement
+	name = "ribbed metal walls"
+	icon = 'icons/turf/walls/hybrisa_offices_colonywall.dmi'
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/office
+	name = "bare metal walls"
+	icon = 'icons/turf/walls/hybrisa_offices_colonywall.dmi'
+	icon_state = "strata_bare_outpost_"
+	desc = "A thick and chunky metal wall. The surface is barren and imposing."
+	walltype = WALL_STRATA_OUTPOST_BARE
+/turf/closed/wall/hybrisa/colony/office/reinforced
+	name = "ribbed metal walls"
+	icon_state = "strata_ribbed_outpost_"
+	desc = "A thick and chunky metal wall covered in jagged ribs."
+	walltype = WALL_STRATA_OUTPOST_RIBBED
+	damage_cap = HEALTH_WALL_REINFORCED
+/turf/closed/wall/hybrisa/colony/office/reinforced/hull
+	hull = 1
```

Comment:
Should use boolean (`TRUE`); you were mostly consistent using booleans, but somereason didn't here.